pom.xml

```diff
@@ -495,30 +495,6 @@ Import-Package: \\
           <version>4.3.1</version>
         </plugin>
 
-        <plugin>
```

Comment:
This change looks rather unrelated to the PR?

Reply:
Yup. reverted.